overlord/snapstate/snapstate.go

```diff
@@ -314,6 +314,27 @@ func refreshRetain(st *state.State) int {
 	return retain
 }
 
+// maxInhibitionTime returns the value of the maximum inhibition time, in seconds
+func maxInhibitionTime(st *state.State) time.Duration {
+
+	var maxInhibitionDays int
+	err := config.NewTransaction(st).Get("core", "refresh.max-inhibition-days", &maxInhibitionDays)
+
+	if err != nil && !config.IsNoOption(err) {
+		logger.Noticef("internal error: refresh.max-inhibition-days system option is not valid: %v", err)
+	}
+
+	// not set, use default value
+	if maxInhibitionDays == 0 {
+		maxInhibitionDays = 14
```

Comment:
This value would probably be appropriate to have as an external constant as it was before (it'd be self-documenting and a bit easier to find)

Reply:
Good point! Done.